HISTORY.md

```diff
@@ -0,0 +1,227 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+
```

Comment:
i think this as to be removed, only the releases should be in here. then gcg should acknowledge this file.

Reply:
we have done this for example in https://github.com/voxpupuli/puppet-wildfly